yarn.lock

```diff
@@ -1028,16 +1028,6 @@
   dependencies:
     "@chakra-ui/theme-tools" "^2.0.2"
 
```

Comment:
Seems weird to me that the `yarn.lock` has been updated, have you rebased your branch to the latest `develop` ? The `yarn.lock`  should stay the same if you didn't add/remove npm packages

Reply:
Reverting the changes :)